package.json

```diff
@@ -63,28 +63,28 @@
     "@stardazed/streams-polyfill": "^2.4.0",
     "@xmtp/proto": "^3.1.0",
     "ethers": "^5.5.3",
-    "long": "^5.2.0",
-    "node-localstorage": "^2.2.1"
+    "long": "^5.2.0"
   },
   "devDependencies": {
     "@commitlint/cli": "^16.1.0",
     "@commitlint/config-conventional": "^16.0.0",
     "@types/bl": "^5.0.2",
     "@types/callback-to-async-iterator": "^1.1.4",
     "@types/jest": "^27.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.31.1",
-    "@typescript-eslint/parser": "^4.31.1",
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
```

Comment:
I was [seeing](https://github.com/xmtp/xmtp-js/actions/runs/3128031452/jobs/5075358977) eslint resolution errors & upgrading resolved the issue.

Reply:
I ran into the same thing and made the same change in one of my PRs. Apologies for the merge conflicts.